webassembly/app.html

```diff
@@ -223,6 +223,54 @@ <h1 class="title">F3D Web</h1>
         // do a first render and start the interactor
         Module.engineInstance.getWindow().render();
         Module.engineInstance.getInteractor().start();
+
+        // Parse search-query mesh url-param or load default mesh file
+        const params = new URLSearchParams(window.location.search);
+        const mesh_url = params.get("mesh");
+        const extension = params.get("extension");
+        if (mesh_url) {
+          fetch(mesh_url)
+          .then((response) => {
+            if (!response.ok) {
+              throw new Error(`HTTP error, status = ${response.status}`);
+            }
+            // Build filename given extension urlparam or response header content-disposition
+            let filename 
```

Comment:
Sorry, I have a C++ background, so let's pretend JS semicolon insertion doesn't exist and add a semicolon after each statement 😬